ai/generic/detect-generic-ai-anthprop.yaml

```diff
@@ -0,0 +1,18 @@
+rules:
+  - id: detect-generic-ai-anthprop
```

Comment:
do we want to correct this typo? It's consistent across all of the tests/files, more of a "polish" issue

Reply:
I had to do this because otherwise the generic matcher matches `anthropic` and then I need a comment on the line above to make the tests patch which leads to an infinite loop of needing more and more comments 😢